[M]34.cpp

```diff
@@ -0,0 +1,48 @@
+// Problem: 34. Find First and Last Position of Element in Sorted Array - Medium
+// Link: https://leetcode.com/problems/find-first-and-last-position-of-element-in-sorted-array/
+
+class Solution {
+public:
+    int findLeft(vector<int>& nums, int target) {
+        int l = 0, r = nums.size() - 1;
```

Comment:
Code thế này là bắt đầu có tư duy rồi đấy :))

Reply:
anh đọc discussion bài này thấy nhiều thằng chỉ cần code 1 hàm search là có thể dùng lại được. Anh code thế này liệu vào pv có bị đánh giá là dài dòng không, có bị hạ điểm ko?

Reply:
Nếu viết 1 hàm dùng chung thì chú phải pass thêm 1 biến (boolean chẳng hạn) rồi if cái biến đó. Theo tiêu chuẩn của clean code thì code như này của chú dễ đọc hơn, dễ maintain hơn vì tên hàm đã nói rõ chức năng, ko cần phải đọc bên trong nhiều, cách kia ko được recommended. A sẽ đánh cách này cao hơn.

Reply:
Thank you, interviewer! :))